neutron/tests/functional/agent/l2/extensions/test_ovs_agent_qos_extension.py

```diff
@@ -29,6 +29,8 @@
 
 TEST_POLICY_ID1 = "a2d72369-4246-4f19-bd3c-af51ec8d70cd"
 TEST_POLICY_ID2 = "46ebaec0-0570-43ac-82f6-60d2b03168c5"
+TEST_DSCP_TAG_1 = 1
```

Comment:
This may not matter functionally, but to prevent confusion on this matter, do we want to avoid illegitimate DSCP marks, particularly odd numbers) and stick with, for instance,  8 and 26, per http://bytesolutions.com/Support/Knowledgebase/KB_Viewer/ArticleId/34/DSCP-TOS-CoS-Presidence-conversion-chart.aspx?